pkg/handler/iam_handler.go

```diff
@@ -219,3 +233,17 @@ func toBindingsMap(bs []*v1alpha1.Binding) map[string]map[string]struct{} {
 	}
 	return result
 }
+
+func expired(exp string) (bool, error) {
+	// An valid expiration expression will look like "request.time < timestamp('2023-06-14T19:55:03Z')".
+	matches := regexp.MustCompile(`request.time < timestamp\('([^']+)'\)`).FindStringSubmatch(exp)
+	// Unable to find enough submatches.
+	if len(matches) < 2 {
+		return false, fmt.Errorf("expression %q does not match format %q", exp, "request.time < timestamp('2006-01-02T15:04:05Z07:00')")
```

Comment:
Is it expected that we write hard-coded timestamp `2006-01-02T15:04:05Z07:00` here?

Reply:
It is just a format example, will update and make it clearer.